snowpack/src/build/import-resolver.ts

```diff
@@ -56,7 +56,7 @@ export function createImportResolver({
   config,
 }: ImportResolverOptions) {
   return function importResolver(spec: string): string | false {
-    if (URL_HAS_PROTOCOL_REGEX.test(spec)) {
+    if (URL_HAS_PROTOCOL_REGEX.test(spec) || config.installOptions.externalPackage?.includes(spec)) {
```

Comment:
ha, this looks like an outdated version of what we use in `build.ts`. Can you replace both snippets with:

```
// Ignore "http://*" imports
if (url.parse(spec).protocol) {
  return spec;
}

// Ignore packages marked as external
if (this.config.installOptions.externalPackage?.includes(spec)) {
    return spec;
}
```

It's definitely a little more verbose, but these functions are notoriously buggy so good documentation is important.

Reply:
Otherwise, LGTM!

Reply:
Sure thing, done!